### PR TITLE
Feature/4790 5221 key submission metadata unit tests

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Some unit tests for key submission metadata unit tests

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5221

## Screenshots
NA

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift [hunk 1]
```diff
@@ -233,7 +233,10 @@ class ExposureSubmissionCoordinator: NSObject, ExposureSubmissionCoordinating, R
 	}
 	
 	private func createTestResultViewController(with testResult: TestResult) -> ExposureSubmissionTestResultViewController {
-		updateStoreWithKeySubmissionMetadataDefaultValues()
+		// store is only initialized when a positive test result is received
+		if testResult == .positive {
+			updateStoreWithKeySubmissionMetadataDefaultValues()
+		}
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:239]: when the store is only initialized when a positive test result is received, what happens if there is a negative result? Do we track then nothing?

File: src/xcode/ENA/ENA/Source/Services/Exposure Submission/ExposureSubmissionService.swift [hunk 2]
```diff
@@ -108,8 +108,7 @@ class ENAExposureSubmissionService: ExposureSubmissionService {
 		}.store(in: &subscriptions)
 	}
 
-	func getTemporaryExposureKeys(completion: @escaping ExposureSubmissionHandler) {
-		Log.info("Getting temporary exposure keys...", log: .api)
+	func getTemporaryExposureKeys(completion: @escaping ExposureSubmissionHandler) {		Log.info("Getting temporary exposure keys...", log: .api)
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Services/Exposure Submission/ExposureSubmissionService.swift:111]: indentation is wrong?

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:239]: Yes. Nothing. 

Tech Specs:
A data structure to represent the values for PPAKeySubmissionMetadata is created when a positive test result is received.

File: src/xcode/ENA/ENA/Source/Services/Exposure Submission/ExposureSubmissionService.swift (hunk 2 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Services/Exposure Submission/ExposureSubmissionService.swift]: Good pick. Done

**Review by pascal-brause — APPROVED** [review]

**Review by 30mar — APPROVED** [review]

**Pull request merged**